composer.json

```diff
@@ -16,7 +16,7 @@
     },
     "require": {
         "php": ">=7.1",
-        "cakephp/cakephp": "^3.6",
+        "cakephp/cakephp": "^3.6|4.x-dev",
```

Comment:
You should never do this. Pretty sure 3.x and 4.x cannot work both here.

Reply:
I've updated it to dev-master for the 4.0.1 fix #14033 related to issue #14032. It can be 4.0.1 after CakePHP/CakePHP 4.0.1 release.